src/Components/Components/src/EventCallbackFactory.cs

```diff
@@ -135,6 +135,23 @@ public string Create<T>(object receiver, string callback)
             return callback;
         }
 
+        /// <summary>
+        /// Returns the provided <paramref name="callback"/>. For internal framework use only.
+        /// </summary>
+        /// <param name="receiver"></param>
+        /// <param name="callback"></param>
+        /// <returns></returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public EventCallback<T> Create<T>(object receiver, EventCallback callback)
```

Comment:
Found which porting this. This allows you to assign an `EventCallback` to an `EventCallback<T>`. This is useful if you want to pass an eventcallback through 3 layers of components - and yes we had a test that hit this 😆